### PR TITLE
feat: filter and product list added

### PR DESCRIPTION
**Pull request by som749898**
Please review my pr

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
please add a screenshot of the product also, it makes it easy even if u have a preview link.
everything looks good.!!

**Comment by poornima-alugubelly** [comment]
I understand that u need the PR to be reviewed but please try to have the description about the changes and screenshot instead s in PR description

**Review by poornima-alugubelly — APPROVED** [review]
looks good!

File: Product/product.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.product-filter h4 {
+  padding-bottom: 5px;
+}
+
+.filter {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (aniketsingh13) [inline Product/product.css:12]: you can make a class for it . if it repeat in your code

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: Product/product.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.product-filter {
+  width: 18.75rem;
+}
+
+.product-filter h4 {
```
Comment (saifudinbadani) [inline Product/product.css:5]: maybe use css combinators it will increase specificity and if possible try to use classes instead of elements.

**Review by saifudinbadani — APPROVED** [review]

**Pull request merged**